apis/submariner/v1alpha1/submariner_types.go

```diff
@@ -120,11 +121,19 @@ func init() {
 func (submariner *Submariner) SetDefaults() {
 	if submariner.Spec.Repository == "" {
 		// An empty field is converted to the default upstream submariner repository where all images live
-		submariner.Spec.Repository = versions.DefaultSubmarinerRepo
+		if versions.OverrideDefaultRepo != "" {
```

Comment:
See my comment down in versions for a simpler approach.

Reply:
Thanks @mangelajo I will get rid of these Override variables